modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/imagestack/v0.1.0.json

```diff
@@ -0,0 +1,142 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/imagestack/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "import": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/atlas/rotationalmatrix/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:ImageStackShape",
+      "@type": "sh:NodeShape",
+      "label": "Image stack",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:ImageStack",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:imageModality",
+              "name": "Image modality",
+              "description": "Modality of the image stack",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:sliceDirection",
+              "name": "Slice Direction",
+              "description": "Slice direction of the image stack",
+              "sh:in": {
+                "@list": [
```

Comment:
The items can be directly added to a "in" array:
"in":["Sagittal", "Axial", "Coronal"]